.TAOS-CI/.gbs.conf

```diff
@@ -0,0 +1,37 @@
+# TAOS-CI: About a default location of the ".gbs.conf" file
+# If TAOS-CI could not import the (1) ./.TAOS-CI/.gbs.conf or
+# 2) ./packaging/.gbs.conf file, it imports the (3) ~/.gbs.conf file of www-data account.
+# Please keep the ./packaging/.gbs.conf file ASAP for customized GitHub repository.
+
+[general]
+# Current profile name which should match a profile section name
+profile = profile.tizen
+tmpdir = /var/tmp
+editor = vim
+packaging_branch = tizen
+workdir = .
+
+[profile.tizen]
+# Common authentication info for whole profile
+# user = <WRITE_YOUR_TIZEN_ID>
+# passwd = <WRITE_YOUR_TIZEN_PASSSWORD>
+# obs = obs.tizen
+
+# https://source.tizen.org/documentation/developer-guide/environment-setup
+# Note that a buildconf file of the last repo (e.g., /var/tmp/{USER}-gbs/tizen.of) is used by default.
+repos = repo.base, repo.unified, repo.extra
```

Comment:
You may need to consider using:
```
repos = repo.base, repo.extra, repo.unified
```
if you want to keep using obs-configuration of repo.unified.

Reply:
Let's keep the existing order at this stage. 😄